skimage/feature/_cascade.pyx

```diff
@@ -87,11 +87,11 @@ cdef struct Stage:
 
     Py_ssize_t first_idx
     Py_ssize_t amount
-    float threshold
+    cnp.float32_t threshold
 
 
 cdef vector[Detection] _group_detections(vector[Detection] detections,
-                                         float intersection_score_threshold=0.5,
+                                         cnp.float32_t intersection_score_threshold=0.5,
```

Comment:
this seems like a behavior change. float used to be float64. wy the change to 32 bit? Is the change to 32 bit correct?

Reply:
In my mind `float` is `float32` and `double` is `float64` in cython, but I am may be mistaken :confused:

Reply:
Cython [documentation](https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#types) confirms that ;)

Reply:
ok my confusion stems from numpy then:

```
In [1]: np.dtype(float)                                                         
Out[1]: dtype('float64')
```
I actually remember having this same discussion with jni and stefanv